### PR TITLE
Add Twenty Forty Eight app

### PR DESCRIPTION
**Pull request by nmuldavin**
### What does this PR do?

Add Twenty Forty Eight app

This app is built and maintained by us, Datadog, although we used an open-source react project

### CODE REVIEW

File: twenty_forty_eight/README.md [hunk 1]
```diff
@@ -0,0 +1,21 @@
+# Twenty Forty Eight
+
+## Overview
+
+Twenty Forty Eight is an easy and fun game you can add to your Datadog dashboards. This fun extension is made possible by [Datadog apps](https://docs.datadoghq.com/developers/datadog_apps) including the ability to extend the user-interface with third party content not supported natively. If you are interested in extending your datadog experience for buisiness or for fun, we encourage you to build or apps of your own.
```
Comment (jsonmercer) [inline twenty_forty_eight/README.md:5]: I would rewrite the second and third sentences to this: "This fun extension is made possible by [Datadog apps] (https://docs.datadoghq.com/developers/datadog_apps), which can enhance the user experience by adding 3rd-party content not supported natively within Datadog. If you are interested in enhancing your Datadog experience, we encourage you to build an app of your own."

File: twenty_forty_eight/README.md [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+
+### How to play
+
+Click on the widget to activate the app, then use your **arrow keys** to move the tiles. Tiles with the same number **merge into one** when they touch. Add them up to reach 2048 and beyond!
+
+## Setup
+
+1. In your Datadog account, navigate to Dashboards. Select the dashboard that you would like to add the Game to, or [create a new dashboard](https://docs.datadoghq.com/dashboards/#new-dashboard).
```
Comment (jsonmercer) [inline twenty_forty_eight/README.md:13]: "Game" doesn't need to be capitalized.

File: twenty_forty_eight/README.md [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+
+2. In the dashboard, click **+Add Widgets** to the right of the dashboard title. Scroll to the right through the widgets, and then drag and drop the **Twenty Forty Eight** widget into the desired position on your dashboard.
+
+## Support
+
+[Contact Datadog support](https://www.datadoghq.com/support/) if you run in to any issues.
```
Comment (jsonmercer) [inline twenty_forty_eight/README.md:19]: "in to" should be "into"

**Review by jsonmercer — CHANGES_REQUESTED** [review]

File: twenty_forty_eight/README.md (hunk 1 above)
Comment (sarina-dd) [inline twenty_forty_eight/README.md:5]:
```suggestion
Twenty Forty Eight is a fun game you can add to your Datadog dashboards. This extension is made possible by [Datadog apps][1] which extends the user-interface with third party content not supported natively. If you are interested in extending your Datadog experience for business or for fun, you can [build out an app or game of your own][1].
```

**Comment by nmuldavin** [comment]
Issues should be addressed thanks for the feedback

**Review by mgarabed — APPROVED** [review]

**Pull request merged**